hazelcast/src/main/java/com/hazelcast/jet/impl/util/ExceptionUtil.java

```diff
@@ -96,23 +82,24 @@ public static boolean isTopologyException(Throwable t) {
      * Called during startup to make our exceptions known to Hazelcast serialization
      */
     public static void registerJetExceptions(@Nonnull ClientExceptionFactory factory) {
-        for (Tuple3<Integer, Class<? extends Throwable>, ExceptionFactory> exception : EXCEPTIONS) {
-            factory.register(exception.f0(), exception.f1(), exception.f2());
-        }
+        factory.register(JET_EXCEPTIONS_RANGE_START, JetException.class, JetException::new);
```

Comment:
6 lines vs 11 with same readability and no warnings produced by compiler... I think it should be ok to include simple boyscout, WDYT?